README.md

```diff
@@ -9,7 +9,25 @@ You can see details of [the project roadmap here](https://napari.org/roadmaps/in
 
 ## contributing
 
-Contributions are encouraged! Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) to get started. Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+Contributions are encouraged! Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+It's nice to be able preview how your contribution will look on the web, so here we provide a brief summary of what's required for a local setup that will allow you to do so.
+Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) for more comprehensive information about contributing documentation.
+
+### quickstart: local setup
```

Comment:
We could also quickly mention the CircleCI here:

```suggestion
It's nice to be able preview how your contribution will look on the web, so here we provide a brief summary of what's required for a local setup that will allow you to do so. Alternatively, our CI set up provides a preview link that shows your changes. Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) for more comprehensive information about contributing documentation.

### Quickstart: local setup
```

Reply:
I think if adding the CI setup note, the subheading should change to just "quickstart". Also, I think it should remain lowercase to match the rest of the README headings.

Reply:
I like the idea of mentioning the easy CI preview somewhere, don't have an opinion where.

Reply:
I mention the CI preview [here](https://github.com/napari/docs/pull/372/commits/f704b39314d9fd8a4aefbdc723c5f519731a5015) and left the subheading as-is (both casing and text)